.mega-linter.yml

```diff
@@ -9,7 +9,7 @@ DISABLE_LINTERS:
   - MARKDOWN_MARKDOWN_LINK_CHECK
   - TYPESCRIPT_STANDARD
   - DOCKERFILE_HADOLINT
-  - CREDENTIALS_SECRETLINT
+    #  - CREDENTIALS_SECRETLINT
```

Comment:
what's with the secretlint? should we delete this line altogether?

if we'll reenable this later, could you add a comment about why it's commented out and when to reinstate it?

Reply:
I wanted to reenable secretlint, but it complained that `docker/nodes/local-validator` folder contains some (test) cerfiticates. Trying to exclude that folder using secretlint-specific directive didn't work, so I left secretlint disabled.